src/WebJobs.Script/Config/NameResolver.cs

```diff
@@ -1,41 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Globalization;
-
-namespace Microsoft.Azure.WebJobs.Script.Config
-{
-    internal class NameResolver : DefaultNameResolver
-    {
-        private readonly Random _rand = new Random();
-
-        public override string Resolve(string name)
-        {
-            if (string.IsNullOrEmpty(name))
-            {
-                return null;
-            }
-
-            // resolve our special tokens
-            switch (name.ToLowerInvariant())
-            {
-                case "rand-guid":
-                    return Guid.NewGuid().ToString();
-                case "rand-int":
```

Comment:
As part of this I'm also removing the rand-int that I added. There really aren't any scenarios for wanting a random int (can you think of one?)